jit/v2/JIT_v2.0.0.yaml

```
@@ -1422,33 +2151,101 @@ paths:
+                        
+                    too many times within a time period results in an error.
+
+                    **NB**: The `errorCode` is not yet standardized by DCSA. The value `7003` is just a "random example".
+                  value:
+                    httpMethod: PUT
+                    requestUri: /v2/vessel-status/085a3207-5e45-49cf-8e1b-f8442beaf545
+                    statusCode: 429
+                    statusCodeText: Too Many Requests
+                    statusCodeMessage: Too many requests to update the Vessel Status has been requested. Please try again in 1 hour
+                    providerCorrelationReference: 4426d965-0dd8-4005-8c63-dc68b01c4962
+                    errorDateTime: '2024-09-04T09:41:00Z'
+                    errors:
+                      - errorCode: 7003
+                        errorCodeText: Max Vessel Status updates reached
+                        errorCodeMessage: A maximum of 100 unique Vessel Status can be updated per hour
   '/v2/vessel-statuses':
     get:
       parameters:
-        - $ref: '#/components/parameters/portCallServiceIDQueryParam'
-
-        - $ref: '#/components/parameters/limitQueryParam'
-        - $ref: '#/components/parameters/cursorQueryParam'
+        - $ref: '#/components/parameters/portCallServiceIDRequiredQueryParam'
         - $ref: '#/components/parameters/Api-Version-Major'
       tags:
         - Port Call Service
       operationId: get-vessel-status
       summary: Retrieves a list of **Vessel Statuses**
       description: |
-        Retrieves a list of **Vessel Statuses**
+        Retrieves a list of **Vessel Statuses** that match the specified filter criteria. It is **mandatory** to provide the `portCallServiceID`
+
+        The resultset of this endPoint will always include **Vessel Statuses** that are linked to **Port Calls** that have **not yet been completed**. Definition of a completed **Port Call** is:
+        ````
+        A Port Call:
+        - that has a Terminal Call
+        - that has a Port Call Service with: `portCallServiceType=BERTH` AND `portCallServiceEventTypeCode=DEPA`
+        - that has a Timestamp with: `classifierCode=PLN` AND `dateTime < {now}`
+        ````
+        This can potentially result in empty resultSets. **Example:** filtering by `portCallServiceID` included in a **Port Call** that has been completed.
+
+        Here is are some examples queries:
+          * To get a list of **Vessel Statuses** for at specific **Port Call Service** use the `portCallServiceID` filter with the `portCallServiceID` of the **Port Call Service** to filter by. This will result in a list of potentially many **Vessel Statuses** all of which will be linked to the **Port Call Service** specified.
+        
+        It is up to the Depending on the implementation of the **Vessel Status** - this can be a list of **Vessel Status** updates or it can be a list with a single object being updated every time the **Vessel Status** is sent.
```

```suggestion
        It is up to, and depending on the implementation of the **Vessel Status** - this can be a list of **Vessel Status** updates or it can be a list with a single object being updated every time the **Vessel Status** is sent.
```
Looks like you're updating this sentence still.